recipes/traja/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "traja" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d13d54d66887b0c2de1db6d5161d72e96f8411364364e6b442e7e5f341f09a5f
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3
+  run:
+    - python >=3
+    - pandas
+    - matplotlib
+    - numpy
+    - psutil
+    - scipy
+    - seaborn
+    - shapely
+
+test:
+  imports:
+    - traja
+    - traja.contrib
+
+about:
+  home: https://github.com/justinshenk/traja
+  license: MIT
+  license_family: MIT
+  license_file: 
```

This should specify the name of the license file in the repository:
```suggestion
  license_file: LICENSE
```

---

Thanks, added!